src/analysis/retail/priest/shadow/modules/talents/MindDevourer.tsx

```diff
@@ -0,0 +1,102 @@
+import Analyzer, { SELECTED_PLAYER, Options } from 'parser/core/Analyzer';
+import Events, { ApplyBuffEvent, RemoveBuffEvent } from 'parser/core/Events';
+import { When, ThresholdStyle } from 'parser/core/ParseResults';
+import AbilityTracker from 'parser/shared/modules/AbilityTracker';
+import EventHistory from 'parser/shared/modules/EventHistory';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+
+// Example Log https://www.warcraftlogs.com/reports/ctvYK32ZhbDqLmX8#fight=30&type=damage-done
+
+class MindDevourer extends Analyzer {
+  static dependencies = {
+    eventHistory: EventHistory,
+    abilityTracker: AbilityTracker,
+  };
+  protected eventHistory!: EventHistory;
+  protected abilityTracker!: AbilityTracker;
+
+  procsGained: number = 0;
+  procsWasted: number = 0;
+  lastProcTime: number = 0;
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasTalent(TALENTS.MIND_DEVOURER_TALENT.id);
+    this.addEventListener(
+      Events.applybuff.by(SELECTED_PLAYER).spell(SPELLS.MIND_DEVOURER_TALENT_BUFF),
+      this.onBuffApplied,
+    );
+    this.addEventListener(
+      Events.removebuff.by(SELECTED_PLAYER).spell(SPELLS.MIND_DEVOURER_TALENT_BUFF),
+      this.onBuffRemoved,
+    );
+  }
+
+  onBuffApplied(event: ApplyBuffEvent) {
+    this.procsGained += 1; // Add a proc to the counter
+    this.lastProcTime = event.timestamp;
+  }
+
+  onBuffRemoved(event: RemoveBuffEvent) {
+    const durationHeld = event.timestamp - this.lastProcTime;
+    if (durationHeld >= 14990) {
+      this.procsWasted += 1;
+    }
+  }
+
+  getProcsUsed() {
+    return this.procsGained - this.procsWasted;
+  }
+
+  get suggestionThresholds() {
+    return {
+      actual: this.procsWasted / this.procsGained,
+      isGreaterThan: {
+        minor: 0,
+        average: 0,
+        major: 0.05,
+      },
+      style: ThresholdStyle.PERCENTAGE,
+    };
+  }
+
+  suggestions(when: When) {
+    when(this.suggestionThresholds).addSuggestion((suggest, actual, recommended) =>
+      suggest(
+        <>
+          You wasted {formatPercentage(actual)}% of{' '}
+          <SpellLink id={TALENTS.MIND_DEVOURER_TALENT.id} /> procs.{' '}
+        </>,
+      )
+        .icon(TALENTS.MIND_DEVOURER_TALENT.icon)
+        .actual(
+          t({
+            id: 'priest.shadow.suggestions.mindDevourer.efficiency',
+            message: `Wasted ${formatPercentage(actual)}% of Mind Devourer procs.`,
+          }),
+        )
+        .recommended(`<${formatPercentage(recommended)}% is recommended.`),
+    );
+  }
+
+  statistic() {
+    return (
+      <Statistic category={STATISTIC_CATEGORY.TALENTS} size="flexible">
+        <BoringSpellValueText spellId={TALENTS.MIND_DEVOURER_TALENT.id}>
+          <>
+            {this.getProcsUsed()}/{this.procsGained} <small>Procs Used</small>
+          </>
```

Comment:
You don't need to wrap this in a fragment as the `BoringSpellValueText` is already wrapping the contents.
```suggestion
            {this.getProcsUsed()}/{this.procsGained} <small>Procs Used</small>
```